index.js

```diff
@@ -70,18 +67,9 @@ class CopyButtonPlugin {
       navigator.clipboard
         .writeText(newText)
         .then(function () {
-          button.innerHTML = locales[lang]?.[1] || "Copied!";
           button.dataset.copied = true;
 
-          let alert = Object.assign(document.createElement("div"), {
-            role: "status",
-            className: "hljs-copy-alert",
-            innerHTML: locales[lang]?.[2] || "Copied to clipboard",
-          });
-          el.parentElement.appendChild(alert);
-
           setTimeout(() => {
-            button.innerHTML = locales[lang]?.[0] || "Copy";
             button.dataset.copied = false;
             el.parentElement.removeChild(alert);
             alert = null;
```

Comment:
should be removed

Reply:
Fixed here: https://github.com/arronhunt/highlightjs-copy/pull/44/commits/58751e90761a76520f986afde4cacd39c25148b0